src/gt4py/next/program_processors/runners/dace/gtir_sdfg.py

```diff
@@ -600,24 +604,29 @@ def _add_sdfg_params(
         self,
         sdfg: dace.SDFG,
         node_params: Sequence[gtir.Sym],
-        symbolic_arguments: set[str],
+        scalars_as_dace_symbols: bool = False,
+        symbolic_arguments: Optional[Iterable[str]] = None,
     ) -> list[str]:
         """
         Helper function to add storage for node parameters and connectivity tables.
 
-        GT4Py field arguments will be translated to `dace.data.Array` objects.
-        GT4Py scalar arguments will be translated to `dace.data.Scalar` objects,
-        except when they are listed in 'symbolic_arguments', in which case they
-        will be represented in the SDFG as DaCe symbols.
+        For details about storage allocation of each data type, see `_add_storage()`.
         """
+        if symbolic_arguments is None:
+            symbolic_arguments = {}
```

Comment:
```suggestion
            symbolic_arguments = set()
```

See also my note in the `visit_Program()` function where this function is called.